src/renderer/components/uielements/addressEllipsis/AddressEllipsis.tsx

```diff
@@ -19,10 +19,11 @@ type Props = {
   className?: string
   enableCopy?: boolean
   linkIcon?: React.ReactElement
+  displayText?: string | React.ReactElement
```

Comment:
What's the reason to extend `AddressEllipsis` instead of using a lable for "SELF" addresses? Or do we need a copy icon for SELF addresses ...?

Reply:
hm. makes sense